Packs/AWS-EC2/ReleaseNotes/1_2_0.md

```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+##### AWS - EC2
+- Internal code improvements.
```

Comment:
can rm. not needed when we have detailed rn. any pr contains also some internal code improvements..